### PR TITLE
feat: [SRM-10261]: Added SLO charts in Change details card

### PR DESCRIPTION
**Pull request by shashidhargg**
##### Summary:

SLO and Error Budget charts in Change Details Card

##### Jira Links:

https://harness.atlassian.net/browse/SRM-10261

##### Screenshots:

Screenshot 2022-05-04 at 10 51 33 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by shashidhargg** [comment]
trigger cypress

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessCDEventCard/HarnessCDEventCard.tsx [hunk 1]
```diff
@@ -44,11 +47,22 @@ export default function HarnessCDEventCard({ data }: { data: ChangeEventDTO }):
...
+      {data.monitoredServiceIdentifier && data.eventTime && (
         <ChangeEventServiceHealth
           monitoredServiceIdentifier={data.monitoredServiceIdentifier}
           startTime={data.eventTime}
           eventType={data.type}
+          timeStamps={timeStamps}
+          setTimestamps={setTimestamps}
+        />
+      )}
+      {data.monitoredServiceIdentifier && data.eventTime && (
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessCDEventCard/HarnessCDEventCard.tsx:59]: merge with the condition check above

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessNextGenEventCard/HarnessNextGenEventCard.tsx [hunk 2]
```diff
@@ -71,12 +71,23 @@ export default function HarnessNextGenEventCard({ data }: { data: ChangeEventDTO
...
+      {data.eventTime && data.monitoredServiceIdentifier && (
         <ChangeEventServiceHealth
           monitoredServiceIdentifier={data.monitoredServiceIdentifier}
           startTime={data.eventTime}
           eventType={data.type}
+          timeStamps={timeStamps}
+          setTimestamps={setTimestamps}
+        />
+      )}
+      {data.monitoredServiceIdentifier && data.eventTime && (
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessNextGenEventCard/HarnessNextGenEventCard.tsx:84]: merge conditions

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessCDEventCard/HarnessCDEventCard.tsx (hunk 1 above)
Comment (shashidhargg) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessCDEventCard/HarnessCDEventCard.tsx]: Done

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessNextGenEventCard/HarnessNextGenEventCard.tsx (hunk 2 above)
Comment (shashidhargg) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/components/ChangeEventCard/components/EventCards/HarnessNextGenEventCard/HarnessNextGenEventCard.tsx:75]: Done

**Pull request closed without merging**